cmd/collectors/restperf/restperf.go

```diff
@@ -1421,6 +1437,17 @@ func (r *RestPerf) pollInstance(records []gjson.Result, apiD time.Duration) (map
 			continue
 		}
 
+		if isWorkloadObject(r.Prop.Query) || isWorkloadDetailObject(r.Prop.Query) {
+			// The API endpoint api/storage/qos/workloads lacks an is_constituent filter, unlike qos-workload-get-iter. As a result, we must perform client-side filtering.
+			// Although the api/private/cli/qos/workload endpoint includes this filter, it doesn't provide an option to fetch all records, both constituent and flexgroup types.
+			if r.perfProp.disableConstituents {
+				if match := constituentRegex.FindStringSubmatch(instanceData.Get("volume").String()); len(match) == 3 {
```

Comment:
slightly more efficient to use MatchString like
`if constituentRegex.MatchString(instanceData.Get("volume").String()) {
continue
}`